ApplicationServices/LibraryCommands.cs

```diff
@@ -18,6 +18,8 @@ public static class LibraryCommands
 
 		public static async Task<List<LibraryBook>> FindInactiveBooks(Func<Account, ILoginCallback> loginCallbackFactoryFunc, List<LibraryBook> existingLibrary, params Account[] accounts)
 		{
+			logRestart();
```

Comment:
glad to see you like my poor man's benchmarker that I forgot to delete :)

Reply:
It was hard to miss when it was throwing exceptions!

Reply:
Oh shit. It didn’t do that to me. I guess I need to push through your fix to a release so that’s not what other users see

Reply:
Yeah, it was crashing on the first logTime because __log was empty, so log..Last() threw an exception. It wouldn't happen if, before launching RemoveBooksDialog, you ran import books at least once.